meg_runtime/locking/lockFile.py

```diff
@@ -0,0 +1,116 @@
+"""MEG system lockfile parser
+
+Can be used to parse the lock file and preform operations on the lockfile
+Only interacts with the lockfile, does not preform any git actions
+Does not check permissions before actions are taken
+
+All file paths are relitive to the repository directory
+Working directory should be changed by the git module
+"""
+
+import json
+import os.path
+import time
+from meg_runtime.logger import Logger
+
+class LockFile:
```

Comment:
You probably want to at least derive from object, but I wonder if dict would be better though? Since you have access to those underscore properties outside this class, it would be better to have this class be a dict itself instead of containing multiple.